workflows/diagnostics/fv3net/diagnostics/offline/_input_sensitivity.py

```diff
@@ -2,73 +2,18 @@
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 import logging
 import numpy as np
-from typing import Tuple, Hashable, Iterable, Dict, Mapping
+from typing import Mapping
 import xarray as xr
 
 import fv3fit
-from fv3fit.sklearn._random_forest import SklearnWrapper
-from fv3fit.keras.jacobian import compute_jacobians, nondimensionalize_jacobians
+
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
-OutputSensitivity = Dict[str, np.ndarray]
-
-
-def dataset_to_dict_input(ds):
-    data = {}
-    for var in ds:
-        # for predictions, drop the 'target' values
-        if "derivation" in ds[var].dims:
-            values = ds[var].sel({"derivation": "predict"}).values
-        else:
-            values = ds[var].values
-        if len(ds[var].dims) == 1:
-            values = values.reshape(-1, 1)
-        data[var] = values
-    return data
-
-
-def _count_features_2d(
-    quantity_names: Iterable[Hashable], dataset: xr.Dataset, sample_dim_name: str
-) -> Dict[Hashable, int]:
-    """
-    count features for (sample[, z]) arrays.
-    Copied from fv3fit._shared.packer, as this logic is pretty robust.
-    """
-    for name in quantity_names:
-        if len(dataset[name].dims) > 2:
-            value = dataset[name]
-            raise ValueError(
-                "can only count 1D/2D (sample[, z]) "
-                f"variables, recieved values for {name} with dimensions {value.dims}"
-            )
-    return_dict = {}
-    for name in quantity_names:
-        value = dataset[name]
-        if len(value.dims) == 1 and value.dims[0] == sample_dim_name:
-            return_dict[name] = 1
-        elif value.dims[0] != sample_dim_name:
-            raise ValueError(
-                f"cannot count values for {name} whose first dimension is not the "
-                f"sample dimension ({sample_dim_name}), has dims {value.dims}"
-            )
-        else:
-            return_dict[name] = value.shape[1]
-    return return_dict
-
-
-def _get_variable_indices(
-    stacked: xr.Dataset, variables: Iterable[Hashable]
-) -> Dict[Hashable, Tuple[int, int]]:
 
-    variable_dims = _count_features_2d(variables, stacked, "sample")
-    start = 0
-    variable_indices = {}
-    for var, var_dim in variable_dims.items():
-        variable_indices[var] = (start, start + var_dim)
-        start += var_dim
-    return variable_indices
+RandomForestInputSensitivity = Mapping[str, Mapping[str, np.ndarray]]
```

Comment:
Missing usage of this ... does it make sense to import `fv3fit.RandomForestInputSensitivity` for use here instead of having to redefine it again here and as `info` below, or does that overly couple this to fv3fit? Seems pretty reasonable I think.

Reply:
Yeah, I think so- since this is already super coupled to fv3fit from having to extract the relevant structured information out of the `InputSensitivity`, if that coupling breaks it'd be easier to debug if the types were imported.